src/Monitors/Models.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Collection\Monitors;
+
+use Collection\Monitors\Monitors;
+use PDO;
+
+class Models
+{
+    private PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+    
+    
+    public function getAllProducts(): array
```

Comment:
getAllProducts? Shouldn't it be getAllMonitors?

Reply:
I agonised a lot over names, one of the simplest things to do but hard for me. Changed name.